discord/permissions.py

```diff
@@ -762,6 +786,9 @@ class PermissionOverwrite:
         use_external_stickers: bool | None
         start_embedded_activities: bool | None
         moderate_members: bool | None
+        view_creator_monetization_analytics: bool | None
```

Comment:
@Lulalaby This can't be an overwrite ? Same as `view_guild_insights` for example

Reply:
Needs to be added to all_channel in that case tho, that I forgot to do.

Reply:
ye add the missing ones